tests/backup/backup_test.go

```diff
@@ -8166,10 +8145,10 @@ func createUsers(numberOfUsers int) []string {
 	log.InfoD("Creating %d users", numberOfUsers)
 	var wg sync.WaitGroup
 	for i := 1; i <= numberOfUsers; i++ {
-		userName := fmt.Sprintf("testuser%v", i)
+		userName := fmt.Sprintf("testuser%v-%v", i, time.Now().Unix())
 		firstName := fmt.Sprintf("FirstName%v", i)
 		lastName := fmt.Sprintf("LastName%v", i)
-		email := fmt.Sprintf("testuser%v@cnbu.com", i)
+		email := fmt.Sprintf("testuser%v@cnbu.com-%v", i, time.Now().Unix())
```

Comment:
Since userName is unique, you can use userName@cnbu.com

Reply:
done!!